src/ui_flow.c

```diff
@@ -113,10 +123,21 @@ UX_STEP_CB(
       strings.common.fullAddress + 26
     });
 
+  UX_STEP_CB(
+    ux_settings_flow_4_step,
+    bnnn,
+    switch_settings_display_fee_details(),
+    {
+      "Fee Details",
+      "Display fee details",
+      "when possible",
```

Comment:
I added "when possible" because when dealing with non EIP1559 transactions, there are no details. But I'm wondering if it wouldn't be better to just remove this sentence.

Reply:
"when available" maybe ?

Reply:
d724d4196ccd34b3fbba5d585cbc21c1706fff01